docs/install/windows-dev.markdown

```diff
@@ -0,0 +1,74 @@
+## Building on Windows
+
+Pre-requisites:
+
+* You'll need to install [Visual Studio](https://visualstudio.microsoft.com/), the community edition works fine. Install the `Desktop development with C++` workload.
+* Install `rustup` to get the `rust` compiler installed on your system.
+  [Install rustup](https://www.rust-lang.org/en-US/install.html). You don't need to install 
+  the `Visual Studio C++ Build tools` if you already installed Visual Studio.
+* Rust version 1.56 or later is required
```

Comment:
This is going to get stale and overlooked when the min version is next bumped.

For unix systems the version is encoded in `ci/check-rust-version.sh`, but since that was written cargo now supports specifying the minimum version in the manifest, so it's probably best to put it somewhere mostly central; perhaps in the `config` crate, which is built fairly early and stands a good chance of alerting the user early when they are using the wrong version.